frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditCheckbox.tsx

```diff
@@ -47,8 +47,8 @@ const transformCheckboxFieldToEditForm = (
 ): EditCheckboxInputs => {
   const nextValidationOptions = field.validateByValue
     ? {
-        customMin: field.ValidationOptions.customMin || '',
-        customMax: field.ValidationOptions.customMax || '',
+        customMin: field.ValidationOptions.customMin?.toString() || '',
```

Comment:
This means we do not need this toString call anymore, nor the parseInt call below.

Reply:
[6744778](https://github.com/opengovsg/FormSG/pull/3702/commits/6744778bc42caf15ca8ddd6ffcf75568869db583)